packages/core/src/shared/index.ts

```diff
@@ -59,3 +59,4 @@ export { i18n } from './i18n-helper'
 export * from './icons'
 export * as textDocumentUtil from './utilities/textDocumentUtilities'
 export { TabTypeDataMap } from '../amazonq/webview/ui/tabs/constants'
+export { LSPDownloader, Manifest, Content } from './downloadLSP'
```

Comment:
Unless I'm mistaken, the `* as foo` pattern is what we want to encourage. That allows us to use very few index.ts files which contain everything, without worrying about name conflicts. It's also more ergonomic for the consumer(?)
```suggestion
export * as fetchLsp from './downloadLSP'
```